common/config/rush/pnpm-lock.yaml

```diff
@@ -1509,6 +1511,11 @@ packages:
       '@opentelemetry/api': 1.0.3
     dev: false
 
+  /@opentelemetry/api-metrics/0.27.0:
```

Comment:
is this file relevant in case of user apps? if so, can it be removed for now? same for instrumentation-api package below

Reply:
This looks like a transitive dependency pulled in by @opentelemetry/instrumentation 

```bash
instrumentation-opentelemetry instrumentation-opentelemetry % npm ls @opentelemetry/api-metrics               
@azure/instrumentation-opentelemetry@1.0.0-beta.1 /home/mleger/workspace/worktrees/tracing/sdk/instrumentation/instrumentation-opentelemetry
└─┬ @opentelemetry/instrumentation@0.27.0 -> /home/mleger/workspace/worktrees/tracing/common/temp/node_modules/.pnpm/@opentelemetry+instrumentation@0.27.0_@opentelemetry+api@1.0.3/node_modules/@opentelemetry/instrumentation
  └── UNMET DEPENDENCY @opentelemetry/api-metrics@0.27.0

npm ERR! missing: @opentelemetry/api-metrics@0.27.0, required by @opentelemetry/instrumentation@0.27.0
```

So that explains why it shows up in lock-file. Unfortunately I don't know if we can yank it out